libr/io/io.c

```diff
@@ -889,25 +794,7 @@ R_API bool r_io_is_blockdevice(RIO *io) {
 }
 
 R_API ut64 r_io_size(RIO *io) {
```

Comment:
Imho iosize should be removed or return the total size of memory allocated. Not just the current file